Mage.Sets/src/mage/cards/t/TrespassersCurse.java

```diff
@@ -100,7 +100,8 @@ public boolean checkTrigger(GameEvent event, Game game) {
         Permanent enchantment = game.getPermanent(this.getSourceId());
         if (enchantment != null
                 && enchantment.getAttachedTo() != null
-                && game.getControllerId(event.getSourceId()).equals(enchantment.getAttachedTo())) {
+                && game.getControllerId(event.getSourceId()).equals(enchantment.getAttachedTo())
+                && game.getPermanent(this.getSourceId()).isCreature()) {
```

Comment:
I'm actually pretty sure that it was working when this PR was merged. I tested it out before submitting. But it looks like it was changed again after my PR was merged :/ https://github.com/magefree/mage/commit/6156ebf5b668a7f6bdf235d3856d3678fcdba3f4